toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -46,7 +46,7 @@ func enableVerityPartition(buildDir string, verity *imagecustomizerapi.Verity, i
 	return true, nil
 }
 
-func addDracutModule(dracutModuleName string, dracutDriverName string, imageChroot *safechroot.Chroot) error {
+func addDracutModuleAndDriver(dracutModuleName string, dracutDriverName string, imageChroot *safechroot.Chroot) error {
```

Comment:
The only difference between `addDracutModuleAndDriver`, `addDracutModule`, and `addDracutDriver` is the content of `lines`. So, it should be fairly straightforward to deduplicate this code.